services/crypto_approve_allowance.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+import "basic_types.proto";
+
+/**
+ * Creates one or more hbar/token approved allowances <b>relative to the payer account of this
+ * transaction</b>. Each allowance grants a spender the right to transfer a pre-determined 
+ * amount of the payer's hbar/token to any other account of the spender's choice. 
+ * 
+ * (So if account <tt>0.0.X</tt> pays for this transaction, then at consensus each spender 
+ * account will have new allowances to spend hbar or tokens from <tt>0.0.X</tt>).
+ */
+message CryptoApproveAllowanceTransactionBody {
+    /**
+     * List of hbar allowances approved by the account owner.
+     */
+    repeated CryptoApproval cryptoApprovals = 1;
+
+    /**
+     * List of token fungible allowances approved by the account owner.
+     */
+    repeated TokenApproval tokenApprovals = 2;
```

Comment:
nit: Sort nft in between crypto and token here and elsewhere.

Reply:
Done